src/app/overview/wallet-overview/wallet-overview.component.ts

```diff
@@ -0,0 +1,175 @@
+import { Component, OnInit } from '@angular/core';
+import { ExplorerModel } from '../../explorer/explorer.model';
+import { ExplorerService } from '../../explorer/explorer.service';
+import { WalletModel } from '../../wallet/wallet.model';
+import { WalletService } from '../../wallet/wallet.service';
+import { RpcSend } from '../../rpc/rpc-send.model';
+import { RpcReceive } from '../../rpc/rpc-receive.model';
+
+export interface SendToAddressModel {
+  amount: Number;
+  destinationAddress: String;
+  feeIncluded: Boolean;
+}
+
+
+@Component({
+  selector: 'app-wallet-overview',
+  templateUrl: './wallet-overview.component.html',
+  styleUrls: ['../overview.component.css','./wallet-overview.component.css']
+})
+export class WalletOverviewComponent implements OnInit {
+  
+
+  explorer: ExplorerModel;
+  wallet: WalletModel;
+  transaction: SendToAddressModel = {
+    amount: undefined,
+    destinationAddress: undefined,
+    feeIncluded: false,
+  };
+  rpcReceive: RpcReceive;
+  qrMainAddress: string;
+
+  constructor(
+    private explorerService: ExplorerService,
+    private walletService: WalletService
+  ) {}
+
+  ngOnInit() {
```

Comment:
The wallet overview needs `showUSD`, `showBTC`, `showBalance` and `getStakeReport`. I don't think it needs any other functionality.

Reply:
This is still true. Make sure you remove any unused inputs or interfaces at the top of this file as well